armips/data/moves.s

```diff
@@ -4026,7 +4026,7 @@ movedata MOVE_WHIRLPOOL, "Whirlpool"
 movedata MOVE_BEAT_UP, "Beat Up"
     battleeffect 154
     pss SPLIT_PHYSICAL
-    basepower DEBUG_NEEDS_TESTING ? 0 : 10 // will beat up still work with base power 0?
+    basepower 5
```

Comment:
does this really have base power 5?

Reply:
![image](https://github.com/BluRosie/hg-engine/assets/135853273/fa2fe398-f66e-4e9a-a6c8-07c9139c853f)
Not 5 exactly, for each mon in the party it's their base attack / 10 with 5 added to it. Like fury cutter, the move's damage calc adjusts the base damage before the main damage calculation

Reply:
what does it show in the summary screen?  this would show 5 i believe--unless i'm wrong and it won't display sub 5

Reply:
You're correct, I didn't think about the summary screen. A quick test shows it works with 0 set, adding 5 in the damage calc.